.github/workflows/on_prerelease.yml

```diff
@@ -195,4 +195,16 @@ jobs:
           integration: nri-${{ env.INTEGRATION }}
           packageLocation: repo
           stagingRepo: true
-          upgrade: false
\ No newline at end of file
+          upgrade: false
+
+  notify-failure:
```

Comment:
you forgot to delete the notification above

Reply:
🤦 Thank you!